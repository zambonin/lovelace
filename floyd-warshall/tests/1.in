4 4
2 3 1
1 2 3
0 1 4
0 3 9
