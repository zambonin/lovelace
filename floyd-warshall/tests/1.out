All pair shortest distances
      0      4      7      8
    inf      0      3      4
    inf    inf      0      1
    inf    inf    inf      0
